run_docker.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
I do not really like filling the root directory. Could this be moved to `ci/`, `build/` or other similarly named directory?